xlive/Blam/Common/Common.h

```diff
@@ -42,7 +42,7 @@ static_assert (sizeof(STRUCT) == _SIZE, "Invalid size for struct (" #STRUCT")")
 static_assert (offsetof(STRUCT, FIELD) == OFFSET, #STRUCT " layout is invalid")
 
 #define TAG_BLOCK_SIZE_ASSERT(tagblock,size)\
-static_assert (sizeof(tagblock) == (size),"Invalid Size for TagBlock <" #tagblock ">")
+static_assert (sizeof(tagblock) == size,"Invalid Size for TagBlock <" #tagblock ">")
```

Comment:
why?

Reply:
Was looking at the assert while working on the definitions and the brackets seemed unnecessary.
I can add them back if you want.

Reply:
macro pitfalls: https://gcc.gnu.org/onlinedocs/gcc-3.0.1/cpp_3.html#SEC26

Reply:
reverted back to before